src/Tooltip/Tooltip.js

```diff
@@ -351,7 +352,9 @@ module.exports = kind(
 		if (this.showing && this.hasNode()) {
 			var b = this.node.getBoundingClientRect(),
 				moonDefaultPadding = 20,
```

Comment:
Wouldn't this be an easier fix than changing all the variables?
```
moonDefaultPadding = ri.scale( 20 ),
```
Also, having "rem" in the name doesn't make sense because we aren't storing "rem" units, but storing scaled pixels instead.